src/libraries/System.IO.Compression.ZipFile/src/System.IO.Compression.ZipFile.csproj

```diff
@@ -18,13 +18,15 @@
   </ItemGroup>
   <!-- Windows specific files -->
   <ItemGroup Condition="'$(TargetPlatformIdentifier)' == 'windows'">
+    <Compile Include="$(CommonPath)System\IO\Compression\ZipArchiveEntryConstants.Windows.cs" />
```

Comment:
This is never used on Windows, correct? I think we can delete this line and just include `ZipArchiveEntryConstants.Unix.cs` on non-Windows

```suggestion
```

Reply:
Oh, good point, thank you @eerhardt